src/main/flight/alt_hold.h

```diff
@@ -29,9 +29,9 @@ typedef struct {
     float targetAltitudeCm;
     float targetAltitudeAdjustRate;
     float measuredAltitudeCm;
+    float altitudeDerivativeCmS;
```

Comment:
Not needed unless you want to manipulate this value but I doubt it.
```suggestion
```

Reply:
yes, done throughout